### DISCUSSION TITLE
Am I still sane?

### DISCUSSION THREAD

**Discussion by Abelkrijgtalles** [Abelkrijgtalles/mojang-maps#15] [General]
This is just a collection of commit messages that make me question if I'm still sane

**Comment by Abelkrijgtalles** [comment]
![image](https://github.com/Abelkrijgtalles/mojang-maps/assets/56084910/3f2a03d7-260f-47f2-b4d3-8ade0e717b10)

**Comment by Abelkrijgtalles** [comment]
![image](https://github.com/Abelkrijgtalles/mojang-maps/assets/56084910/8409b169-a9fe-44f5-9a96-f9b6a1802fd0)

**Comment by Abelkrijgtalles** [comment]
![image](https://github.com/Abelkrijgtalles/mojang-maps/assets/56084910/88356f5f-7b27-415b-8511-c13d64bc309a)

**Comment by Abelkrijgtalles** [comment]
![image](https://github.com/user-attachments/assets/26f7d9c9-8449-4b36-86d5-ddf0b7ccfb28)

**Comment by samuelh2005** [comment]
Those commit messages are actually quite normal for GitHub users that do not follow a convention.
